src/templates/finops-hub/modules/hub.bicep

```diff
@@ -32,6 +32,11 @@ param convertToParquet bool = true
 
 @description('Optional. Enable telemetry to track anonymous module usage trends, monitor for bugs, and improve future releases.')
 param enableDefaultTelemetry bool = true
+@description('Optional. To use Private Endpoints, add target subnet resource Id.')
+param subnetResourceId string = ''
+
+@description('Optional. To disable Public Network Access, set to "Disabled".')
+param publicNetworkAccess string = ''
```

Comment:
Out of curiosity, is there a reason why someone would specify a subnetResourceId and _not_ disable public network access? What about disable public network access without specifying a subnetResourceId? Just curious if it would make sense to remove the network access option and have everything work off the subnetResourceId parameter. 

(Sorry, I don't have any context on this, so it may be obvious to others 😓)

Reply:
+1 , if a user provides a subnetResourceId then we should assume that they want to disable public access and use this param as the condition